router.go

```diff
@@ -10,8 +11,27 @@ import (
 	"k8s.io/kubernetes/pkg/util"
 )
 
+// Do we want to run nginx without kubernetes driving configuration?
+var kubeless bool
+
+func init() {
+	const (
```

Comment:
not really, just followed the [Go example for pkg flag](https://golang.org/pkg/flag/#example_) which used this convention so I went with it.